### DISCUSSION TITLE
Can aiortc be used to implement SFU meetings?

### DISCUSSION THREAD

**Discussion by laoyin** [aiortc/aiortc#969] [Q&A]
Can aiortc be used to implement SFU meetings?

**Comment by GuDuJian-J-Zhang** [comment]
> Can aiortc be used to implement SFU meetings?

yes i have implemented one

**Comment by laoyin** [comment]
@GuDuJian-J-Zhang  thanks,   i founded some pc`s method and attribute ,    i will try

**Comment by laoyin** [comment]
@GuDuJian-J-Zhang  i just try to addTrack, but  javascript  can not get the "track" event?   how do you solve this problem?  should i  renegotiate?

**Comment by diversees** [comment]
Did you resolve this? Why is it that addTrack() does not trigger negotiationneeded event? Looks like that event is not in aiortc.
